low-level/sspl-ll.spec

```diff
@@ -78,7 +78,9 @@ cp -p $SSPL_SETUP/consuldump.py $SSPL_BASE/low-level/
 # take backup of cache folder if exists
 mkdir -p /opt/seagate/backup/%{version}
```

Comment:
This backup folder should stay with in /opt/seagate/cortx

Also backup should limit to /var/%{product_family}/sspl/**data**, target folder being /opt/seagate/cortx/sspl/data, if its meant to retain just persistence folder & sspl.conf too, why sspl folder?

Reply:
This is legacy. But I agree with your point.
I will bring the changes.

Reply:
Done.

> [root@ssc-vm-3120 ~]# ls -lrt /opt/seagate/cortx/backup/2.0.0/sspl/
> total 12
> drwxr-xr-x 5 sspl-ll sspl-ll 4096 Apr  6 13:24 data
> -rw-r--r-- 1 root    root    4176 Apr  6 13:34 sspl.conf